sdks/python/apache_beam/options/pipeline_options_validator_test.py

```diff
@@ -479,20 +470,17 @@ def test_experiment_region_and_worker_region_mutually_exclusive(self):
     self.assertIn('experiment', errors[0])
     self.assertIn('worker_region', errors[0])
 
-  def test_experiment_region_and_worker_zone_mutually_exclusive(self):
+  def test_region_and_worker_zone_mutually_exclusive(self):
     runner = MockRunners.DataflowRunner()
     options = PipelineOptions([
-        '--experiments',
```

Comment:
Did we lose some coverage of behavior with experiments here?

Reply:
Nope, the region and zone flags are no longer experiments. We were testing the failure expected when declaring both region and zone. I just updated to use the flags(modern usage), not use the flags declared as experiments (out of date usage).